pkg/bundle/prompt.go

```diff
@@ -232,9 +232,70 @@ func getOutputDir(t *templatecache.TemplateData) error {
 	return nil
 }
 
-func getExistingParamsPath(in string) (string, error) {
-	prompt := promptui.Prompt{
-		Label: fmt.Sprintf("Path to an existing %s to generate params from, leave blank to skip", in),
+func getExistingParamsPath(templateName string) (string, error) {
+	prompt := promptui.Prompt{}
+
+	switch templateName {
```

Comment:
Not a huge fan of a switch statement for this, but alternatives will likely require a deeper redesign that will take much more time than allotted.